### ISSUE TITLE
Earth dmg ignores def.

### ISSUE THREAD

**Issue by JaydonTheWarrior** [Wynncraft/Issues#7361]
As stated earth dmg ignores def, as shown in the screenshots provided.

**Comment by DebugMC** [comment]
It does not, i tested this.

Also refer to #6905 #7108 they COULD  be involved.

**Comment by DebugMC** [comment]
Oh btw does this work on players and mobs or just one?

**Comment by Moe-Ronickah** [comment]
@DebugMC , obviously, from his first image, earth damage was by-passing earth defense, as Tourotos have defense against every element.

This will have to wait until their is a code update for earth damage.
